assets/dev/js/editor/components/settings/editor-preferences/manager.js

```diff
@@ -52,6 +53,12 @@ export default class extends BaseManager {
 		$link.attr( 'media', 'auto' === newValue ? '(prefers-color-scheme: dark)' : '' ).appendTo( elementorCommon.elements.$body );
 	}
 
+	onPanelWidthChanged( newValue ) {
+		elementor.panel.saveSize( { width: newValue.size + newValue.unit } );
+
+		elementor.panel.setSize();
```

Comment:
It's already set, no?

Reply:
@kobizz

Reply:
No. This method just applying the size to the panel width

Reply:
Can you rename it to applySize?

Reply:
This is an old method. We can't deal with backward compatibility at this stage